docs/core/testing/unit-testing-with-nunit.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Unit testing C# with NUnit and .NET Core
+title: Unit testing C# with NUnit
```

Comment:
Same problem

![image](https://user-images.githubusercontent.com/31348972/71553839-d3432580-2a1e-11ea-87d3-7a8a56d4e787.png)

I won't point to other occurrences of it to reduce the number of comments.